module/pom.xml

```diff
@@ -162,7 +167,31 @@
     </plugins>
   </build>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.mule</groupId>
+        <artifactId>mule-javaee-runtime-bom</artifactId>
```

Comment:
Is this really needed? I would expect dependency management to be in the parent pom. Here, in the project, I would use the dependency directly.

Reply:
removed that